test/rep.spec.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env mocha */
+const StringHelper = require('../index')
+const expect = require('chai').expect
+
+describe('#rep', () => {
+  it('appends by default one time the string if no `repetitions` is sent', () => {
+    let helper = new StringHelper('ones: ')
+    let result = helper.rep('1').str()
+    expect(result).to.equal('ones: 1')
+  })
+
+  it('convertes to a number 1 the `rep` argument if not a number', () => {
```

Comment:
If you mean 'converts' you have an extra 'e' but if you meant 'convierte' your are missing an 'i' and still having an extra 's', whatever the case is, you know what I mean.